rusk/src/lib/chain.rs

```diff
@@ -27,6 +27,7 @@ pub const MINIMUM_STAKE: Dusk = dusk(1000.0);
 pub struct RuskTip {
     pub current: [u8; 32],
     pub base: [u8; 32],
+    pub epoch: Option<[u8; 32]>,
```

Comment:
Does this need to be option? We always have an epoch

Reply:
This is for backwards compatibility reasons. When starting up the node, the epoch commit is currently not present on disk. Ergo the use of `Option`.